v2/config/default/manager_pod_disruption_budget.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: policy/v1
```

Comment:
Rather than put this in `default` can you put it in `v2/config/manager/manager_pod_disruption_budget.yaml`, and add that file to the `kustomization.yaml`?

Reply:
Done!